tests/AnchoredViewTest.js

```diff
@@ -471,7 +471,7 @@ describe('AnchoredView', () => {
     it("sets fallback flag to true if passes", async () => {
       const rotationTarget = '0xAbcdef0123456789000000000000000000000005';
       let encoded = encodeRotationMessage(rotationTarget);
-
+      // sign rotation message with wrong key
```

Comment:
My bad - this is actually not relevant in this test, was copied for the other one https://github.com/compound-finance/open-oracle/pull/115/files/bc6a11f3c082713f348be8382a35fcf86354a38c#diff-95c902c9b4b043b32ae9c8358534d27cR474. How about I'll just take over your branch with my changes on top?